spec/lib/service_provider_spec.rb

```diff
@@ -211,10 +212,10 @@
 
   describe '#block_encryption' do
     context 'when no value is specified in YAML' do
-      it 'returns "none"' do
```

Comment:
should we maybe just change this to `returns ServiceProvider::DEFAULT_ENCRYPTION` to make it more evergreen?